fusermount: add page (#5582)
tldr-pages/tldr: pages/common/fusermount.md

# fusermount

> > Mount and unmount FUSE filesystems.

Reviewer: I can´t find an example for mount using this command :thinking:
Maybe we should update the command description only for `unmount`.

Author: There is a `-o` option which expects mount options, so it should at least be possible to mount filesystems. But I couldn't find those options anywhere.

Author: Maybe those are the same options for `mount`, but that's just an asumption.

Reviewer 2: The page doesn't need to have all possible examples, so we can leave it for someone else to pick up later.

Reviewer 2: I think so.  It might even prompt a reader to say "why not?" and then fix it. 🤷🏻

> More information: <https://man.archlinux.org/man/fusermount.1>.

- Unmount a FUSE filesystem:

`fusermount -u {{path/to/mount_point}}`

> - Unmount a FUSE filesystem, as soon as it's not busy anymore:

Reviewer 2: ```suggestion
- Unmount a FUSE filesystem as soon as it becomes unused:
```

`fusermount -z {{path/to/mount_point}}`

> - Print the version of `fusermount` and exit:

Reviewer: ```suggestion
- Display version:
```
